http-handler/src/main/java/com/flipkart/poseidon/handlers/http/impl/HttpConnectionPool.java

```diff
@@ -324,7 +324,13 @@ public HttpRequestBase createHttpRequest(String uri, byte[] data, Map<String, St
             setRequestBody(request,data);
             setRequestHeaders(request, requestHeaders);
             return request;
-        } else
+        } else if ("PATCH".equals(requestType))
```

Comment:
here PATCH definition is according to all the other http methods definition in the same file.